cmd/snap/cmd_pack.go

```diff
@@ -72,6 +73,8 @@ func init() {
 			"check-skeleton": i18n.G("Validate snap-dir metadata only"),
 			// TRANSLATORS: This should not start with a lowercase letter.
 			"filename": i18n.G("Output to this filename"),
+			// TRANSLATORS: This should not start with a lowercase letter.
+			"compression": i18n.G("Compression to use"),
```

Comment:
I'm not sure if this help string is going to be shown anywhere given that the flag is _hidden_, but should it hint at allowed compression methods, e.g. "Compression to use (xz, gzip)" (with xz as the only one for now)? Maybe the list of supported methods could be declared in a list and picked up dynamically here and below when args are validated.

Reply:
Nice, added!